Sources/Support/DebugUI/SDKHealthStatus+Icon.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  SDKHealthStatus+Icon.swift
+//
+//  Created by Pol Piella on 4/10/25.
+
+import SwiftUI
+
+@available(iOS 16.0, macOS 13.0, *)
+extension PurchasesDiagnostics.SDKHealthStatus {
+    var icon: some View {
+        switch self {
+        case let .healthy(warnings):
+            Image(systemName: warnings.count > 0 ? "checkmark.circle.badge.questionmark.fill" : "checkmark.circle.fill")
+                .foregroundColor(.green)
+        case .unhealthy:
+            Image(systemName: "exclamationmark.triangle")
```

Comment:
Not my expertise, but since an `unhealthy` status is actually meaning an error that must be fixed, shouldn't we use here `"xmark.circle.fill" (the same as `PurchasesDiagnostics.SDKHealthCheckStatus` is using for the `failed` case)

Reply:
100%! Great catch, I ended up using what we had before, but having the x asset makes a lot more sense, on ittt!!